core/nuxeo-operation.js

```diff
@@ -303,6 +304,12 @@ import './nuxeo-connection.js';
       return this._isView(input) && input.selectAllActive;
     }
 
+    _hasExcludedItems(input) {
+      return input.selectAllActive && input._excludedItems && input._excludedItems.length > 0
+        ? input._excludedItems
+        : [];
```

Comment:
it could be made shorter and safer with:
```suggestion
      return input.selectAllActive && Array.isArray(input._excludedItems)
        ? input._excludedItems
        : [];
```